newsfragments/542.feature.rst

```diff
@@ -1 +1 @@
-* Reworked :mod:`trio` namespace construction, making it more understandable by static analysis tools. This should improve tab completion in editors, reduce false positives from pylint, and is a first step towards providing type hints.
\ No newline at end of file
+Reworked :mod:trio, :mod:trio.testing, and :mod:trio.socket namespace construction, making them more understandable by static analysis tools. This should improve tab completion in editors, reduce false positives from pylint, and is a first step towards providing type hints.
```

Comment:
You need the backticks around the module names :-) It's a restructured text thing: http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#roles

```
Reworked :mod:`trio`, :mod:`trio.testing`, and :mod:`trio.socket` ...
```